Project.toml

```
@@ -64,50 +64,50 @@ TrixiMakieExt = "Makie"
 TrixiNLsolveExt = "NLsolve"
 
 [compat]
-Accessors = "0.1.12"
+Accessors = "0.1.36"
 CodeTracking = "1.0.5"
-ConstructionBase = "1.3"
+ConstructionBase = "1.5"
 Convex = "0.16"
 DataStructures = "0.18.15"
 DelimitedFiles = "1"
 DiffEqBase = "6 - 6.143"
-DiffEqCallbacks = "2.25"
+DiffEqCallbacks = "2.35"
 Downloads = "1.6"
 ECOS = "1.1.2"
 EllipsisNotation = "1.0"
-FillArrays = "0.13.2, 1"
+FillArrays = "1.9"
 ForwardDiff = "0.10.24"
 HDF5 = "0.16.10, 0.17"
 IfElse = "0.1"
 LinearAlgebra = "1"
 LinearMaps = "2.7, 3.0"
 LoopVectorization = "0.12.151"
 MPI = "0.20.6"
-Makie = "0.19, 0.20"
+Makie = "0.21"
 MuladdMacro = "0.2.2"
 NLsolve = "4.5.1"
 Octavian = "0.3.21"
-OffsetArrays = "1.12"
+OffsetArrays = "1.13"
 P4est = "0.4.12"
 Polyester = "0.7.10"
-PrecompileTools = "1.1"
+PrecompileTools = "1.2"
 Preferences = "1.3"
 Printf = "1"
-RecipesBase = "1.1"
-RecursiveArrayTools = "2.38.10"
+RecipesBase = "1.3.4"
+RecursiveArrayTools = "3.27.1"
 Reexport = "1.0"
 Requires = "1.1"
-SciMLBase = "1.90, 2"
+SciMLBase = "2.67.0"
 SimpleUnPack = "1.1"
 SparseArrays = "1"
 StableRNGs = "1.0.2"
-StartUpDG = "0.17.7, 1.1.5"
+StartUpDG = "1.1.5"
 Static = "0.8.7"
```

```suggestion
Static = "0.8.7, 1"
```

The major change was the removal of pointer arithmetic.

---

Yeah it will need this, and likely the `DiffEqBase` upgrade.